modules/webbrowser/data/js/inviwoapi.js

```diff
@@ -162,6 +162,22 @@ class InviwoAPI {
             }
         });
     }
+    /*
+     * Call a previously registered processor callback.
+     * @param callback  name of the callback
+     * @param data      payload will be stringified into JSON
```

Comment:
Why not in JSON format already? 
Would make it more like the other functions.

Reply:
doesn't really matter, does it? If you put in a string, it stays a string. Putting in an object transforms it to JSON.